lib/src/entrypoint.dart

```diff
@@ -226,6 +226,38 @@ class Entrypoint {
     bool dryRun = false,
     bool precompile = false,
   }) async {
+    // We require an SDK constraint lower-bound as of Dart 2.12.0
```

Comment:
Should we move this to general validation of the pubspec? I guess that would make a difference for consuming 1.0 packages, but that doesn't really work anyway.

Reply:
> I guess that would make a difference for consuming 1.0 packages, but that doesn't really work anyway.

Except all tests have SDK version `0.1.2+4`, so it might further break those :see_no_evil:

Reply:
Also if you try to depend on a Dart 1.x package, you won't a bug that it failed to parse the `pubspec`, instead you'll get a `SolveFailure`.

More importantly, we still need to understand Dart 1.x packages, because they still exist on pub.dev. So we need to be able to read Dart 1.x packages without an SDK constraint.

Reply:
Good points. I just wonder if we want to do the check for other entrypoints?

But I guess they mostly don't work before a pub get has run anyways.